recipes/cad1/task1/baseline/enhance.py

```diff
@@ -247,6 +261,25 @@ def process_stems_for_listener(
     return processed_stems
 
 
+def clip_and_save(signal: np.ndarray, filename: Path | str, config: DictConfig) -> None:
```

Comment:
Wouldn't it be better to just have a function to clip and return a result which is then saved. This seems to be merging two tasks. 

I would also avoid passing config dictionaries. Clearer and more explicit to just provide a soft_clip and sample_rate parameter.

Reply:
I put both tasks together as all audios are clipped and saved. But, this can be split into 2 functions and perhaps move them to `clarity/utils`